addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -223,6 +223,17 @@ var SnippetEditor = Widget.extend({
         }
         return _t("Block");
     },
+    isUpToDate: function () {
+        const targetClass = [...this.$target[0].classList].filter(x => /\bs_./g.test(x))[0];
+        if (!targetClass) {
+            return true;
+        }
```

Comment:
I would check that all of our up-to-date snippet do have that kind of class and return `false` in this case, not `true`.